transport/tchannel/channel_outbound.go

```diff
@@ -156,10 +156,13 @@ func (o *ChannelOutbound) Call(ctx context.Context, req *transport.Request) (*tr
 		return nil, toYARPCError(req, err)
 	}
 
-	// Inject tracing system baggage
-	reqHeaders := tchannel.InjectOutboundSpan(call.Response(), req.Headers.Items())
+	reqHeaders := req.Headers.Items()
+	if o.transport.exactCaseHeader {
+		reqHeaders = req.Headers.OriginalItems()
+	}
 
-	if err := writeRequestHeaders(ctx, format, reqHeaders, call.Arg2Writer); err != nil {
+	tracingBaggage := tchannel.InjectOutboundSpan(call.Response(), nil)
```

Comment:
can you add a comment that we don't want to canonicalize these headers and put the reason